planet/order_request.py

```diff
@@ -79,8 +79,8 @@ def build_request(name: str,
         details['notifications'] = notifications
 
     if order_type:
-        order_type = specs.validate_order_type(order_type)
-        details['order_type'] = order_type
+        validated_order_type = specs.validate_order_type(order_type)
+        details['order_type'] = validated_order_type
```

Comment:
In general, mypy discourages reassigning names, and not without reason because it can be a source of bugs.

Reply:
Oh! yeah, I can see that. nice catch.

Reply:
I reassign names a lot and it's a hard habit to break :laughing: